source/upgrade.txt

```diff
@@ -72,6 +72,14 @@ Version 8.1.0 Breaking Changes
   creating the collection or database
   implicitly.
 
+  .. note::
+
+     Calling the ``EnsureCreated()`` or ``EnsureCreatedAsync()`` method throws
+     an error if your application is running on a
+     :atlas:`shared cluster </reference/free-shared-limitations/>`.
```

Comment:
just making sure this should be "shared cluster" and not "sharded cluster" (assuming so, from the link)

Reply:
Yeah it's supposed to be "shared"